src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -558,17 +571,31 @@ public void subscribeToAttributePath(
         minInterval,
         maxInterval,
         false,
-        false);
+        false,
+        imTimeoutMs);
   }
 
-  /** Subscribe to the given event path */
+  /**
+   * @brief Auto-Resubscribe to the given event path with keepSubscriptions and isFabricFiltered
+   * @param SubscriptionEstablishedCallback Callback when a subscribe response has been received and
+   *     processed
+   * @param ResubscriptionAttemptCallback Callback when a resubscirption haoppens, the termination
+   *     cause is provided to help inform subsequent re-subscription logic.
+   * @param ReportCallback Callback when a report data has been received and processed for the given
+   *     paths.
+   * @param devicePtr connected device pointer
+   * @param eventList a list of event paths
+   * @param imTimeoutMs im interaction time out value, it would override the default value in c++ im
```

Comment:
1. Require to add minInterval, maxInterval.
2. Require to delete ResubscriptionAttemptCallback
3. eventList -> eventPath

Reply:
done, thanks